src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -123,6 +123,85 @@
     'Page',
 ]
 
+WEBPROD = 'fourfront-webprod'
+WEBPROD_TWO = 'fourfront-webprod2'
+MASTERTEST = 'fourfront-mastertest'
+HOTSEAT = 'fourfront-hotseat'
+WEBDEV = 'fourfront-webdev'
+
+
+BEANSTALK_PROD = [
+    WEBPROD,
+    WEBPROD_TWO,
+]
+
+BEANSTALK_TEST = [
+    MASTERTEST,
+    HOTSEAT,
+    WEBDEV,
+]
+
+FF_DEPLOY_CONFIG_SIZE = 2
+
+class DeploymentConfig(object):
+    """
+        Basic class wrapping an arbitrary number of configuration options
+        The caller must know the structure and how to interpret it
+        Values default to None and are set to None when del'd
+    """
+
+    def __init__(self, size):
+        self.tuple = [None] * size
+        self.size = size
+
+    def __getitem__(self, idx):
+        if (idx >= self.size):
+            raise IndexError('Index out of bounds on DeploymentConfig.get')
+        return self.tuple[idx]
+    
+    def __setitem__(self, idx, val):
+        if (idx >= self.size):
+            raise IndexError('Index out of bounds on DeploymentConfig.set')
+        self.tuple[idx] = val
+
+    def __delitem__(self, idx):
+        if (idx >= self.size):
+            raise IndexError('Index out of bounds on DeploymentConfig.del')
+        self.tuple[idx] = None
+
+
+def get_my_env(app):
+    """
+    Gets the env name of the currently running environments
+    """
+    return app.registry.settings.get('env.name')
+
+def get_deployment_config(app):
```

Comment:
Except for the use of positional arguments (addressed above, I'd rather use keyword calling as less fragile, but I won't for now, unless asked, point to all the places that are implicated), this structure looks a _lot_ better. I do find myself wondering, though,